lib/Twig/Environment.php

```diff
@@ -448,6 +450,8 @@ public function resolveTemplate($names)
      */
     public function clearTemplateCache()
     {
+        @trigger_error(sprintf('%s is deprecated and will be removed in Twig 2.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
The Symfony convention for deprecation messages is to say ``The method __METHOD__ is deprecated since X.Y and will be removed in Z.0. Use ... instead.`` (the second sentence being removed if there is no alternative). What do you think about being consistent in Twig ?